discord.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://discord.gg/KfpqwZB">
+  <script>location="https://discord.gg/KfpqwZB"</script>
+  <meta http-equiv="refresh" content="0; url=https://discord.gg/KfpqwZB">
+  <meta name="robots" content="noindex">
+  <h1>KAMI Blue Discord</h1>
+  <a href="https://discord.gg/KfpqwZB">Click here if you are not redirected.</a>
+    <meta content="https://kamiblue.org/favicon.png" property="og:image">
```

Comment:
Can you remove this? None of the other pages use an image embed

Reply:
I don't want to remove it. There are people/browsers with their redirects disabled.

Reply:
I was talking about the image embed?

Reply:
This isn't just for discord. There are other sites that look for that tag.

Reply:
Okay, then can you add it to the default.html layout instead?

Reply:
Its already there.